.github/workflows/pythonbuild.yml

```diff
@@ -109,6 +109,42 @@ jobs:
         with:
           path: ./wheelhouse/*.whl
 
+  build_wheels_aarch64:
+    needs: [test_python]
+    name: Build wheels on aarch64 for ${{matrix.python_tag}}
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        python_tag: [ "cp35-*", "cp36-*", "cp37-*", "cp38-*", "cp39-*"]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'true'
+
+      - uses: actions/setup-python@v1
+        name: Install Python 3.8
+        with:
+          python-version: "3.8"
+
+      - name: Install cibuildwheel
+        run: |
+          python -m pip install cibuildwheel~=1.8.0
+
+      - uses: docker/setup-qemu-action@v1
+        name: Set up QEMU
+
+      - name: Build wheel
+        run: |
+          python -m cibuildwheel --output-dir wheelhouse
+        env:
+          CIBW_ARCHS_LINUX: aarch64
+          CIBW_BUILD: ${{matrix.python_tag}}
```

Comment:
Would this work for the other build platforms of cibuildwheel that I currently skip as well (ppc64le, s390x)

Reply:
I have put x86_64, aarch64, ppc64le, s390x  architectures under one matrix. Please have a look and let me know if you have any concerns.